tests/bluetooth/tester/testcase.yaml

```diff
@@ -14,8 +14,66 @@ tests:
     platform_allow:
       - rd_rw612_bga
       - frdm_rw612
+      - frdm_mcxw71
     extra_configs:
       - CONFIG_NXP_MONOLITHIC_BT=n
+  bluetooth.general.tester_gatt_long_dev_name:
+    build_only: true
+    harness: bluetooth
+    platform_allow:
+      - frdm_mcxw71
+    extra_configs:
+      - CONFIG_BT_CONN_DISABLE_SECURITY=y
+      - CONFIG_BT_DEVICE_NAME_MAX=64
+      - CONFIG_BT_DEVICE_NAME="TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTT"
+  bluetooth.general.tester_enforce_mitm:
+    build_only: true
+    harness: bluetooth
+    platform_allow:
+      - frdm_mcxw71
+    extra_configs:
+      - CONFIG_BT_SMP_ENFORCE_MITM=y
```

Comment:
change to 
```
extra_configs:
    platform:frdm_mcxw71:CONFIG_BT_SMP_ENFORCE_MITM=y
```
and you can remove the `platform_allow`

Reply:
I'm still not quite following. Wouldn't `ENFORCE_MITM=y` be required on any platform that this test is run on?

Reply:
you are right, actually just remove the `platform_allow` will be ok